src/__tests__/CitySearch.test.js

```diff
@@ -55,6 +55,13 @@ describe('<CitySearch /> component', () => {
         expect(CitySearchWrapper.state("suggestions")).toEqual(filteredLocations);
     });
 
+    //NEWLY ADDED TEST
+    test('when the suggestion input does not contain any character', async () => {
+        const eventObject = { target: { value: '' } };
```

Comment:
ok let's use a value that is different than `berlin` or `london` a value that will make the suggestions equal to `[]`.
- maybe '111'